src/content/guides/getting-started.md

```diff
@@ -171,7 +171,7 @@ __dist/index.html__
    </head>
    <body>
 -    <script src="./src/index.js"></script>
-+    <script src="main.js"></script>
++    <script src="./dist/main.js"></script>
```

Comment:
`main.js` is in `dist`, So as `index.html`. Therefore the previous path is the one we should have here IMO